README.md

```diff
@@ -1,13 +1,23 @@
 # Mario Kart 64
+This work-in-progress decompilation of Mario Kart 64 pursues historical and educational elements within the game found via taking it apart and putting it back together. Inspiration to do so not only emanates from the game's hardware and technology but also its immensely positive effects on the cultures and families of nearly every nationality. See [progress](#Progress) for more information.
 
-```diff
-- Note -
+This repository does not contain assets. Compiling requires asset extraction from a prior copy of the game.
 
-The decomp has altered its approach to calculating progress.
+Currently it only supports these releases: US (us), EU 1.0 (eu-1.0) and EU 1.1 (eu-final).
 
-The progress script is still a work-in-progress and the values below are subject to change.
-This notice will be removed when the changes are complete
-```
+It builds the following ROMs:
```

Comment:
Consider merging this line and the one that precedes it together

`It supports and builds the following...`